### PR TITLE
fix(ME-2080): close command after ssh session is closed

### PR DESCRIPTION
**Pull request by btoonk**
# Description

close command after ssh session is closed
Fixes # (ME-2080)

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Tested locally, with client portal, ssh, sftp and ansible.

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: internal/process/setup.go [hunk 1]
```diff
@@ -8,12 +8,10 @@ import (
 	"os"
 	"runtime"
 	"syscall"
-
-	"go.uber.org/zap"
 )
 
 // SetupUserAndGroups changes the uid, gid, and groups of the current process.
-func SetupUserAndGroups(logger *zap.Logger, params *Parameters) error {
+func SetupUserAndGroups(params *Parameters) error {
```
Comment (adrianosela) [inline internal/process/setup.go:14]: keep the logger parameter, if you remove it you also have to remove it from setup_windows.go (where we do use it)

**Review by adrianosela — CHANGES_REQUESTED** [review]
just a small comment -- this won't build for windows when releasing the CLI. Other than that, looks good.

File: internal/process/setup.go (hunk 1 above)
Comment (btoonk) [reply internal/process/setup.go:14]: I removed it for windows also, this does not work anyway.  This input/output of this function is directly parsed to the sftp client. Not the connector process.

**Review by adrianosela — APPROVED** [review]

File: internal/ssh/server_unix.go [hunk 2]
```diff
@@ -160,7 +163,7 @@ func execCmd(s ssh.Session, cmd exec.Cmd, uid, gid uint64, username string) {
 
 		wg.Wait()
 		cmd.Wait()
-
+		s.Exit(cmd.ProcessState.ExitCode())
```
Comment (adrianosela) [inline internal/ssh/server_unix.go:166]: maybe run in a defer?

File: internal/ssh/server_unix.go [hunk 3]
```diff
@@ -110,9 +112,10 @@ func execCmd(s ssh.Session, cmd exec.Cmd, uid, gid uint64, username string) {
 		cmd.Wait()
 
 		if cmd.ProcessState == nil {
-			cmd.Process.Signal(syscall.SIGHUP)
+			cmd.Process.Signal(syscall.SIGKILL)
 		}
 
+		s.Exit(cmd.ProcessState.ExitCode())
```
Comment (adrianosela) [inline internal/ssh/server_unix.go:118]: maybe run in a defer?

**Pull request merged**